crypto/provider_conf.c

```diff
@@ -238,8 +238,30 @@ static int provider_conf_load(OSSL_LIB_CTX *libctx, const char *name,
         /* Override provider name to use */
         if (strcmp(confname, "identity") == 0)
             name = confvalue;
-        else if (strcmp(confname, "soft_load") == 0)
-            soft = 1;
+        else if (strcmp(confname, "soft_load") == 0) {
+            if ((strcmp(confvalue, "1") ==0)
```

Comment:
(nit) missing space after `==` on most, but not all, of the new lines

Reply:
fixed